tensorflow/lite/experimental/micro/examples/micro_speech/CMSIS/arm_cmplx_mag_squared_q10p6.c

```diff
@@ -0,0 +1,141 @@
+/* This file is a modification of the ARM CMSIS library file arm_cmplx_mag_squared_q15.c
```

Comment:
To keep things as clean as possible, we may need to move these modified CMSIS files and the Apollo3 sources into a separate repo that we download. We have fairly strict automatic checks on copyright, so even though this seems like a reasonable thing to do from the license terms, it's hard to have multiple differing copyright notices in the project.